cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,385 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/types"
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+)
+
+const MAXTARGETSHOW = 10
+
+func contextTargetsSize(disruption v1beta1.Disruption) ([]string, error) {
+	spec := disruption.Spec
+	labels := spec.Selector.String()
+	level := spec.Level
+	// If node level, this \n will give us empty namespaces for node which is intended because nodes
```

Comment:
Is "\n" what we'll get for `ObjectMeta.Namespace` if level is node? Sorry, I'm just confused by this comment

Reply:
this was an old comment when I was slicing out the namespace, it is now no longer a thing because I get the namespace from the disruption so I'm deleting this comment, thanks for the catch.